.travis.yml

```diff
@@ -5,24 +5,23 @@ os:
 osx_image: xcode8.3
 solution: MoreLinq.sln
 mono: 5.0.1
-dist: trusty
+dist: xenial
```

Comment:
Any reason to prefer xenial instead of bionic?

Reply:
None in particular except that xenial was the least required version.